### DISCUSSION TITLE
Record to local folder while using Proxy mode

### DISCUSSION THREAD

**Discussion by Tabarane** [bluenviron/mediamtx#1202] [Questions] [answered]
Hi, currently im working with 360 Camera , which is Garmin Virb 360 , while attempted to re-encode it and save to local file like other cameras, FFMPEG has alot of issue of droped frames , buffer , and overall nightmare, non stable, randoming disconnect etc

but if im using proxy mode , everything works like magic, my only hope is can i issue recording for this transmission being in proxy mode

**Comment by aler9** [comment, marked as answer]
Hello, instructions on how to save streams to disk are available in the README and works for both published streams and proxied streams:

https://github.com/aler9/rtsp-simple-server#save-streams-to-disk

I'm getting convinced into integrating a recording feature into the server, the main issue consists in supporting all the available codecs and picking a format able to do so. The feature is being tracked here: #1399

[a comment by an automated account is not shown]
